fedimint-server/src/net/api.rs

```diff
@@ -259,7 +260,11 @@ impl ConsensusApi {
             }
         }
 
-        Ok(self.get_config_with_sig(dbtx).await)
+        self.get_config_with_sig(dbtx)
+            .await
+            .ok_or(ApiError::server_error(
+                "Client signature not ready".to_string(),
+            ))
```

Comment:
note to self: We do not commit the dbtx in case an error is returned, so the counter will only count successful retrievals.